api/src/domain/updateSidebarCards.test.ts

```diff
@@ -127,6 +127,45 @@ describe("updateRoadmapSidebarCards", () => {
   });
 
   describe("welcome card", () => {
+    const operatingPhases: OperatingPhaseId[] = [
+      "GUEST_MODE",
+      "GUEST_MODE_OWNING",
+      "NEEDS_TO_FORM",
+      "NEEDS_TO_REGISTER_FOR_TAXES",
+      "FORMED_AND_REGISTERED",
+    ];
+
+    it.each<OperatingPhaseId | undefined>(operatingPhases)(
+      "removes welcome card when the updated visibleSidebarCards do NOT have welcome card",
```

Comment:
I'm not sure "removes" is the right word here because in the setup here, `visibleSidebarCards` is an empty array. So nothing is being removed. Same in test below

Reply:
Sure, we're more testing that it's allowed to be empty in this state, rather than constantly re-filled with the initial value like in the bug case.